tests/unit/core/TestThemePreview.js

```diff
@@ -1,8 +1,6 @@
 import { createInternalAddon } from 'core/reducers/addons';
 import themeAction, { getThemeData } from 'core/themePreview';
```

Comment:
hey @bobsilverberg  - from what I can tell this `getThemeData` is connected to installing and getting the theme data if it's installed. I think the only place we can remove this is from the preview pieces (which i think i've done already)

The thing I am confused about is `data-browsertheme` prop itself - I wanted to see this to be sure but I don't see where that traces back to...

Reply:
Ok, I think we need to figure out exactly what that does. I would suggest experimenting a bit. For example, try removing that tag and see how it affects the site when run locally. 

If it turns out we do need to keep the `getThemeData` function for something other than previews, I would suggest moving it somewhere other than the core/themePreview.js file, or renaming that file as you suggest.

Reply:
I can shed some light on this. This function needs a node with a `data-browsertheme` attribute. See https://github.com/mozilla/addons-frontend/blob/85a7e0b30e206eef4a1e712afc4133186385f81b/src/core/themePreview.js#L3 

This is related to the legacy api that handles installation and previewing of lightweight themes.

Here's a snippet of code that shows how it works a bit better https://developer.mozilla.org/ru/docs/Themes-840092-dup/Lightweight_themes. It's in the Russian translation since this content has been removed from the english page a while back it would seem.

Reply:
ok thanks for clarification @muffinresearch. so we'll leave this in place for now where used for install purposes

i'll rename the themePreview file..